src/components/file/DropzoneHelper.tsx

```diff
@@ -0,0 +1,77 @@
+import { useContext, useRef } from 'react';
+
+import FolderOutlinedIcon from '@mui/icons-material/FolderOutlined';
+import UploadFileOutlinedIcon from '@mui/icons-material/UploadFileOutlined';
+import { Button, Stack, Typography } from '@mui/material';
+
+import { useBuilderTranslation } from '@/config/i18n';
+import { BUILDER } from '@/langs/constants';
+
+import { UppyContext } from './UppyContext';
+
+const DropzoneHelper = (): JSX.Element => {
+  const { t } = useBuilderTranslation();
+  const { uppy } = useContext(UppyContext);
+  const ref = useRef<HTMLInputElement>(null);
+
+  const handleClick = () => {
+    const { current } = ref;
+    if (current) {
+      current.click();
+    }
+  };
+
+  const handleFiles = () => {
+    const { current } = ref;
+    if (current) {
+      const { files } = current;
+      if (files) {
+        // add files selected to uppy, this will upload them
+        [...files].map((file) =>
+          uppy?.addFile({ data: file, name: file.name }),
```

Comment:
Add a comment specifying the need to add the `name` key in order to send the file name correctly.